packages/core/src/auth/auth.ts

```diff
@@ -502,8 +506,10 @@ export class Auth implements AuthService, ConnectionManager {
     private async validateConnection<T extends Profile>(id: Connection['id'], profile: StoredProfile<T>) {
         const runCheck = async () => {
             if (profile.type === 'sso') {
+                getLogger().info(`Validating SSO connection ${id}`)
                 const provider = this.getTokenProvider(id, profile)
                 if ((await provider.getToken()) === undefined) {
+                    getLogger().info(`Connection ${id} is not valid`)
```

Comment:
Almost always, the variable part should go at the end of the message. That helps readability.

```suggestion
                    getLogger().info(`Connection is not valid: ${id}`)
```